playbooks/preflight-check.yml

```diff
@@ -118,3 +118,12 @@
     - name: Duplicate flavors exist
       fail: msg="Duplicated flavor IDs are detected."
       when: flavor_check.stdout != ""
+
+- name: Verify openstack_domain 
```

Comment:
Can probably drop preflight as it'll just cause extra noise in failing and won't really matter since we inject it later on anyways.